src/python/platforms/fuchsia/util/fuzzer.py

```diff
@@ -206,7 +206,12 @@ def start(self, fuzzer_args):
       else:
         fuzzer_args.append('-jobs=1')
     self.device.ssh(['mkdir', '-p', self.data_path('corpus')])
-    if [x for x in fuzzer_args if not x.startswith('-')]:
+    # If all the arguments are prepended with '-', then we do not need to add
+    # a corpus.
+    # This list comprehension returns a list of all arguments that do *not*
+    # start with '-'.
+    # Thus, if the list is empty, we append data/corpus/.
+    if not [x for x in fuzzer_args if not x.startswith('-')]:
```

Comment:
why was this needed? we should always be passing a corpus dir, even if it's empty.

Reply:
there is an option to pass in a custom corpus directory. if no corpus directory is passed in we assume you're using the default